newTab.appcache

```diff
@@ -0,0 +1,26 @@
+CACHE MANIFEST
```

Comment:
You provably want a sensible `#version 1.x` comment, so we can nuke the cache as needed using versioning. Also, we should consider using Service Workers to cache all these things. Will file a separate bug.